cmd/poller/collector/collector.go

```diff
@@ -304,8 +304,7 @@ func (c *AbstractCollector) Start(wg *sync.WaitGroup) {
 	defer wg.Done()
 	defer func() {
 		if r := recover(); r != nil {
-			c.Logger.Error().Stack().Err(errs.New(errs.ErrPanic, "")).
-				Msgf("Collector panicked %s", r)
+			c.Logger.Error().Stack().Err(errs.New(errs.ErrPanic, "")).Any("err", r).Msg("Collector panicked")
```

Comment:
stack()?

Reply:
Yes, when handling panics the stack is sometimes helpful